### ISSUE TITLE
Keyboard navigation

### ISSUE THREAD

**Issue by davidoskky** [FreeTubeApp/FreeTube#375]
It would be nice to be able to enable keyboard navigation in the video selections.
This way one can open the subscriptions menu and use the arrows to choose the desired video to watch.

**Comment by PrestonN** [comment]
This can technically be done by tabbing through videos in the rewrite, so I'll consider this implemented.  Any further in this would be attempting to make a TV based interface.

**Comment by fsondej** [comment]
Also vim-like navigation using j and k would be great.

**Comment by manuee** [comment]
+1 Love the idea. Keyboard navigation is fantastic for accessibility (and avoiding RSI issues!)

**Comment by merspieler** [comment]
I'd like to see an improvement on this too... hitting `<Tab>` a dozen times, isn't what I'd call keyboard navigation.

I could imagine, apart from using arrow key for video selection, to use `<Alt>-<0-n>` for selecting subscription playlist etc...

Best thing would be to have controls like [qutebrowser](https://github.com/qutebrowser/qutebrowser) but I think, that might be a bit out of scope.

**Comment by ChunkyProgrammer** [comment]
#1525 should solve this issue

**Comment by jasonhenriquez** [comment]
#1525 admittedly does not have left/right arrow navigation of video results, but it does wrap video titles (and all other kinds of results when relevant) in headings, which screen readers have a key for navigating through easily. It shouldn't be too hard for me to add that feature for all users though.

**Comment by 0x6e656b6f** [comment]
I'd like to add my suggestion for keyboard navigation. We can use [Vimium](https://github.com/philc/vimium) as an example for how to open a video using keyboard. Also add way to skip one frame forward and backward using [.] and [,] like on YT.

**Comment by PietJanHein** [comment]
Hotkeys for changing channels and selecting videos would be a great addition to FreeTube.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
@PietJanHein best to comment this on #251

**Comment by PietJanHein** [comment]
> @PietJanHein best to comment this on #251

I will but that is about changing hotkeys and my comment and this topic are about adding hotkeys for navigation in the app. At least based on the topictitle.
